integration-tests/contract-big-map.spec.ts

```diff
@@ -204,7 +209,7 @@ CONFIGS().forEach(({ lib, rpc, setup, knownBigMapContract }) => {
       done();
     });
 
-     it('Return undefined when BigMap key is not found', async (done) => {
+    it('Return undefined when BigMap key is not found', async (done) => {
```

Comment:
It seems like there are duplicated tests in this file and the `integration-tests/contract-big-map-access-by-index.spec.ts` file

Reply:
sorry can't find any duplicated tests in those two files